pallets/handles/src/lib.rs

```diff
@@ -485,11 +492,11 @@ pub mod pallet {
 		/// * `Option<MessageSourceId>` - The `MessageSourceId` if the handle is valid.
 		///
 		pub fn get_msa_id_for_handle(display_handle: Handle) -> Option<MessageSourceId> {
-			let full_handle_str = core::str::from_utf8(&display_handle).unwrap_or("");
-			let (base_handle_str, suffix) = HandleConverter::split_display_name(full_handle_str);
-			let canonical_handle_str = HandleConverter::convert_to_canonical(&base_handle_str);
-			let canonical_handle = canonical_handle_str.as_bytes().to_vec();
-			let canonical_handle: Handle = canonical_handle.try_into().unwrap();
+			let display_handle_str = core::str::from_utf8(&display_handle).unwrap_or("");
```

Comment:
To follow the pattern from above.
```suggestion
			let display_handle_str = core::str::from_utf8(&display_handle).unwrap_or_default();
```